File: openapi-generator/php-ze-ph/src/App/DTO/InlineObject16.php
<?php
declare(strict_types=1);

namespace App\DTO;

use Articus\DataTransfer\Annotation as DTA;

/**
 */
class InlineObject16
{
    /**
     * @DTA\Data(field="externalIds", nullable=true)
     * TODO check validator and strategy are correct and can handle container item type
     * @DTA\Validator(name="Collection", options={"validators":{
     *     {"name":"Type", "options":{"type":"string"}}
     * }})
     * @var map[string,string]
     */
    public $external_ids;
    /**
     * Harsh Event Detection Setting * 0: Passenger * 1: Light Truck * 2: Heavy * 3: Off * 4: Automatic
     * @DTA\Data(field="harsh_accel_setting", nullable=true)
     * @DTA\Validator(name="Type", options={"type":"int"})
     * @var int
     */
    public $harsh_accel_setting;
    /**
     * Name
     * @DTA\Data(field="name", nullable=true)
     * @DTA\Validator(name="Type", options={"type":"string"})
     * @var string
     */
    public $name;
}

File: openapi-generator/dart-jaguar/lib/model/inline_object16.dart
import 'package:jaguar_serializer/jaguar_serializer.dart';

part 'inline_object16.jser.dart';

class InlineObject16 {
  
  @Alias('externalIds')
  final Map<String, String> externalIds;
   /* Harsh Event Detection Setting * 0: Passenger * 1: Light Truck * 2: Heavy * 3: Off * 4: Automatic */
  @Alias('harsh_accel_setting')
  final int harshAccelSetting;
   /* Name */
  @Alias('name')
  final String name;
  

  InlineObject16(
      

{
     this.externalIds = const {},  
     this.harshAccelSetting = null,  
     this.name = null 
    
    }
  );

  @override
  String toString() {
    return 'InlineObject16[externalIds=$externalIds, harshAccelSetting=$harshAccelSetting, name=$name, ]';
  }
}

@GenSerializer(nullableFields: true)
class InlineObject16Serializer extends Serializer<InlineObject16> with _$InlineObject16Serializer {

}
